pkgs/development/python-modules/google-cloud-monitoring/default.nix

```
@@ -9,20 +9,34 @@
 , pytestCheckHook
 , pytest-asyncio
 , mock
+, pythonOlder
 }:
 
 buildPythonPackage rec {
   pname = "google-cloud-monitoring";
-  version = "2.8.0";
+  version = "2.9.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.6"; 
```

https://github.com/NixOS/nixpkgs/runs/5371661373?check_suite_focus=true

```suggestion
  disabled = pythonOlder "3.6";
```

---

Thanks